### PR TITLE
Remove comments builder

### PR DESCRIPTION
**Pull request by stsewd**
PR on top of #31

Related PR: https://github.com/rtfd/readthedocs.org/pull/3802

What is missing?

- [ ] Remove css classes
- [x] Remove related js
- [ ] Re-minify js/css

### CODE REVIEW

**Comment by stsewd** [comment]
The CSS classes have the sphinx prefix `div.sphinx-comments`, those were only used for the comments builder and can be removed, right?

File: readthedocs_ext/_static/javascript/client.js [hunk 1]
```diff
@@ -1,11 +1,2 @@
...
-events = require('./lib/events')
-display = require('./lib/display')
-
 $(document).ready(function () {
-	events.initEvents();
-	comm.initOptions();
-	comm.initMetaData();
-	display.initDisplay();
 })
-
```
Comment (stsewd) [inline readthedocs_ext/_static/javascript/client.js]: All this file can be removed, I suppose. But the `lib/settings.js` was required by the others libs, so it was included in the bundle. Settings.js have information that is used outside the comments builder?

Comment (stsewd) [reply readthedocs_ext/_static/javascript/client.js]: Also, we need to re-minify the js (css too?) after the PR is completed.

**Comment by stsewd** [comment]
I think this PR wasn't ready to be merged, there were some decisions to be made

https://github.com/rtfd/readthedocs-sphinx-ext/pull/38#issuecomment-379091951
https://github.com/rtfd/readthedocs-sphinx-ext/pull/38#discussion_r179618990

**Comment by ericholscher** [comment]
* I believe the entire `sphinxweb.css` can be removed
* I believe all the javascript in `_static` can be removed, except for `readthedocs-data.js_t`

**Pull request merged**